checker/cic.mli

```diff
@@ -312,6 +312,8 @@ type mutual_inductive_body = {
 
     mind_private : bool option; (** allow pattern-matching: Some true ok, Some false blocked *)
 
+    mind_checked_positive : bool; (** [false] when the mutual-inductive was assumed to be well-founded, bypassing the positivity checker.  *)
```

Comment:
I feel very uncomfortable with this comment which I think is a source of misunderstanding, because it suggests that the corresponding (co)inductive types can be assumed as well-founded, even when not checked so, while they can be in practice completely arbitrary, and in particular not at all well-founded.

(and similarly for the comment attached to const_checked_guarded).